src/de/dkfz/b080/co/aceseq/ACESeqMethods.groovy

```diff
@@ -113,7 +113,7 @@ public final class ACESeqMethods {
     public static Tuple2<TextFile, TextFile> mergeDelly(TextFile knownSegmentsFile) {
         TextFile svFile = (TextFile) BaseFile.constructManual(TextFile.class, knownSegmentsFile, null, null, null, null, "dellyFileTag", null, null);
         svFile.setAsSourceFile();
-        JobResult result = new JobResult(knownSegmentsFile.getExecutionContext(), null, JobDependencyID.getFileExistedFakeJob(knownSegmentsFile.getExecutionContext()), false, null, null, null);
+        JobResult result = new JobResult(new BEJobResult(knownSegmentsFile.getExecutionContext(), null, new FakeBEJob(new BEFakeJobID(BEFakeJobID.FakeJobReason.FILE_EXISTED)), null, null, null, null))
```

Comment:
Don't get it. BEJobResult is in the BE core right? It should not take the context as an argument! Why is this here?

Reply:
You are right. The value is actually dropped and another constructor is used. I remove the constructor with the first "Object" type parameter and fix this position.

Reply:
Actually, we could go even further. RoddyCore JobResult is now not more than a call to super() in the constructor. We could completely get rid of it.

Reply:
O.k. JobResult is deleted (see #2).